File: news.js
 <script>
        fetch("https://apinp.com/news/api.php")
            .then(response => {
                if (!response.ok) {
                    throw new Error("Network response was not ok");
                }
                return response.json();
            })
            .then(data => {
                console.log(data);
                document.body.innerHTML = "<pre>" + JSON.stringify(data, null, 2) + "</pre>";
            })
            .catch(error => console.error("Error fetching data:", error));
    </script>

File: news.kt
import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import okhttp3.OkHttpClient
import okhttp3.Request
import org.json.JSONArray

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val client = OkHttpClient()
        val request = Request.Builder()
            .url("https://apinp.com/news/api.php")
            .build()

        Thread {
            client.newCall(request).execute().use { response ->
                if (!response.isSuccessful) throw Exception("Request failed")
                val jsonData = JSONArray(response.body?.string())
                Log.d("API_RESPONSE", jsonData.toString())
            }
        }.start()
    }
}